shared/common-adapters/name-with-icon.stories.js

```diff
@@ -12,22 +12,22 @@ const commonProps = {
 
 const outerClick = evt => {
   if (!evt.defaultPrevented) {
-    action('Outer click')(evt)
+    Sb.action('Outer click')(evt)
   }
 }
 
 const innerClick = evt => {
   evt.preventDefault()
-  action('Inner click')(evt)
+  Sb.action('Inner click')(evt)
 }
 
-const provider = createPropProvider(
-  PropProviders.Avatar(['chrisnojima'], ['chrisnojima']),
-  PropProviders.Usernames(['cecileb', 'chrisnojima'], 'akalin')
+const provider = Sb.createPropProviderWithCommon(
```

Comment:
if common is always included, is there a need to have `WithCommon` in the name?

Reply:
Yes as the decorators actually don't nest so this thing basically does what it used to. It makes a new prop provider and mixes in common to the params